staking/integration-tests/src/integrity_pool/instructions.rs

```diff
@@ -49,13 +49,19 @@ use {
 
 pub fn advance(svm: &mut LiteSVM, payer: &Keypair, publisher_caps: Pubkey) -> TransactionResult {
     let pool_config = get_pool_config_address();
-    let pool_data = fetch_account_data::<PoolConfig>(svm, &pool_config).pool_data;
+    let PoolConfig {
+        pool_data,
+        pyth_token_mint,
+        ..
```

Comment:
This is good syntax. We should use it for SetupResult too instead of renaming to _.